react/features/toolbar/components/Toolbar.native.js

```diff
@@ -1,41 +1,39 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { View } from 'react-native';
 import { connect } from 'react-redux';
 
 import { MEDIA_TYPE, toggleCameraFacingMode } from '../../base/media';
 import { Container } from '../../base/react';
 import { ColorPalette } from '../../base/styles';
+import { beginRoomLockRequest } from '../../room-lock';
 
-import { AbstractToolbar, _mapStateToProps } from './AbstractToolbar';
+import {
+    _mapDispatchToProps as _abstractMapDispatchToProps,
+    _mapStateToProps as _abstractMapStateToProps
+} from './AbstractToolbar';
```

Comment:
If there will no longer be an abstract toolbar, this file shouldn't exist and we should use some functions.js